tests/_bootstrap.php

```diff
@@ -42,3 +41,17 @@
 // By default, do not enable the Custom Tables v1 implementation in tests.
 putenv( 'TEC_CUSTOM_TABLES_V1_DISABLED=1' );
 $_ENV['TEC_CUSTOM_TABLES_V1_DISABLED'] = 1;
+
+function tec_tickets_tests_fake_transactions_enable() {
+	uopz_set_return( Order::class, 'start_transaction', true, false );
```

Comment:
I understand the need to control the transaction process, but the Order class should not have transaction-related methods like `start_transaction`. `rollback_transaction` and `commit_transaction`. They are just passthroughs to the `DB` class, that, defining those magically with `__callStatic`, makes mocking them impossible.
This is a case where I think production code is being affected by testing needs.
The solution is not to mock them.

Create a test case dedicated to this that will:
* run `commit` on `setUp`
* run the test and note the affected post IDs
* clean up the affected post IDs on `tearDown`.